KaoriMiyazono.json

```diff
@@ -0,0 +1,11 @@
+{
+	"type" : 0,
+	"english-name" : "Miyazono Kaori ",
+	"image" : "https://vignette.wikia.nocookie.net/shigatsu-wa-kimi-no-uso/images/3/3d/Ep03.jpg/revision/latest/scale-to-width-down/300?cb=20150306235014",
+	"japanese-name" : "宮園 かおり ",
+	"year" : ,
```

Comment:
This is invalid JSON, I recommend removing this line if you don't know the year because ``year`` is an optional variable.